### PR TITLE
MES-4239: remove code 78 for category B+E

### PR DESCRIPTION
**Pull request by MarkPatt89**
## Description
- Remove `code78` component from `pass-finalisation.cat-be`
- Added warning banner so that selecting automatic transmission still informs DE of licence type to be issued
- Remove redundant tests
- Update reducers and selectors to use `PassCompletion` from common schema
- Bump `mes-test-schema`
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
Screenshot 2019-12-06 at 14 06 33

### CODE REVIEW

File: src/modules/tests/pass-completion/pass-completion.cat-be.reducer.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
 
 import * as passCompletionActions from './pass-completion.actions';
-import { CatBEUniqueTypes } from '@dvsa/mes-test-schema/categories/BE';
+import { PassCompletion } from '@dvsa/mes-test-schema/categories/common';
 import { createFeatureSelector } from '@ngrx/store';
```
Comment (TomBillingtonUK) [inline src/modules/tests/pass-completion/pass-completion.cat-be.reducer.ts]: If there is no difference between Cat B and B+E could we not move stuff into common and get rid of the category specific reducers and selectors

Comment (MarkPatt89) [reply src/modules/tests/pass-completion/pass-completion.cat-be.reducer.ts]: We sure can

**Review by rossfauldsbjss — APPROVED** [review]
looks good

**Review by TomBillingtonUK — APPROVED** [review]
Awesome Job

**Pull request merged**